src/Chainweb/Utils.hs

```diff
@@ -1155,3 +1159,10 @@ threadDelayRng = unsafePerformIO (Prob.createSystemRandom >>= newMVar)
 approximateThreadDelay :: Int -> IO ()
 approximateThreadDelay d = withMVar threadDelayRng (approximately d)
                            >>= threadDelay
+
+data PactCmdLog
```

Comment:
Maybe move this to the logging site? Utils is more for things that are more generally useful.